packages/ajv-validator/package.json

```diff
@@ -14,7 +14,7 @@
     "publint": "attw --pack && publint",
     "typecheck": "tsc -b tsconfig.json",
     "test": "vitest",
-    "version:jsr": "yarn version:set $(yarn version:get)"
+    "version:jsr": "yarn version:set $npm_package_version"
```

Comment:
Reading the documentation, I learned that yarn sets a `$npm_package_version` [environment variable](https://yarnpkg.com/advanced/lifecycle-scripts#environment-variables), so I've used that and removed the `version:get` script